src/controllers/users.controller.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const usersService = require('../services/user.service');
+
+const getAll = (req, res) => {
+  res.send(usersService.getAllUsers());
+};
```

Comment:
for now it is okay, because users is always exist, but in a real project you should handle error here, because you will take users from db etc.

Reply:
Yeah, I just in the middle of implementing the next app with PostgreSQL